### DISCUSSION TITLE
Integracja poprzez Webhook

### DISCUSSION THREAD

**Discussion by opennas** [hackerspace-silesia/Smogomierz#134] [Ideas]
Prosta integracja Smogomierza na dedykowany url byłaby poprzez zastosowanie Webhook.
Wysłane tak dane w realtime mogą być w dowolny sposób analizowane, monitorowane i obrabiane.
Jest wiele możliwości postawienia rozwiązania na webhook i też sporo możliwości łatwych automatyzacji z takiego źródła.

Próbował ktoś webhook? jak możnaby to najłatwiej zrealizować ?

**Comment by bfaliszek** [comment]
Smogomierz wyświetla dane pomiarowe w formie JSON na podstronie IP_Smogomierza/api można z nich w dowolny sposób korzystać.
